api/copy_chunk_experimental.md

```diff
@@ -18,8 +18,24 @@ do not use any experimental features in production.
 |`source_node`|NAME|Data node where the chunk currently resides|
 |`destination_node`|NAME|Data node where the chunk is to be copied|
 
+### Required settings
+
+For `copy_chunk` to work, the `wal_level` must be set to `logical` or
+higher on data nodes. Also consider increasing `max_wal_senders` and
```

Comment:
on data nodes that are going to act as source datanodes for the move/copy activity. Don't need it on the destination datanode.